src/Router/index.ts

```diff
@@ -28,18 +28,16 @@ export interface RouterConfig {
 
 export class Router<M extends DatabaseSchema> {
     interceptorConfig: InterceptorConfig<M>;
-    routerConfig: RouterConfig;
 
     constructor(
       interceptorConfig: Partial<InterceptorConfig<M>> = interceptorDefaultConfig,
-      routerConfig: RouterConfig = { strategies: [] }
+      public routerConfig: RouterConfig = routerDefaultConfig
     ) {
       this.interceptorConfig = merge(
         {},
         interceptorDefaultConfig,
         interceptorConfig
       );
-      this.routerConfig = merge({}, routerDefaultConfig, routerConfig);
```

Comment:
So, this was basically merging two lists together, which I don't think what we want.
So, if user defines `routerConfig` with just fetch, this line will merge two arrays and result will be `fetch` + `xhr`